cli/args/flags.rs

```diff
@@ -576,7 +576,6 @@ pub struct UnstableConfig {
   // TODO(bartlomieju): remove in Deno 2.5
   pub legacy_flag_enabled: bool, // --unstable
   pub bare_node_builtins: bool,
-  pub detect_cjs: bool,
```

Comment:
Just to double check - passing `--unstable-detect-cjs` won't have any effect and will not result in an error?

Reply:
It's still there.